src/thorlabs_cube/driver/kcube/kpa.py

```diff
@@ -0,0 +1,483 @@
+import asyncio
+import struct as st
+
+from thorlabs_cube.driver.base import _Cube
+from thorlabs_cube.driver.message import MGMSG, Message, MsgError
+
+_CHANNEL: int = 0x01
+_RESERVED: int = 0x00
+QUAD_LOOP_PARAMS_SUB_ID: int = 0x01
+QUAD_READINGS_SUB_ID: int = 0x03
+QUAD_POSITION_DEMAND_PARAMS_SUB_ID: int = 0x05
+QUAD_OPER_MODE_SUB_ID: int = 0x07
+QUAD_STATUS_BITS_SUB_ID: int = 0x09
+QUAD_DISP_SETTINGS_SUB_ID: int = 0x0B
+QUAD_POSITION_DEMAND_OUTPUTS_SUB_ID: int = 0x0D
+QUAD_LOOP_PARAMS2_SUB_ID: int = 0x0E
+QUAD_KPA_TRIGIO_SUB_ID: int = 0x0F
+QUAD_KPA_DIGOPS_SUB_ID: int = 0x10
+
+
+class Kpa(_Cube):
+    """KPA101 driver implementation."""
+
+    def __init__(self, loop: asyncio.AbstractEventLoop, serial_dev: str) -> None:
+        """Initialize the KPA101 driver.
+
+        :param loop: Event loop for asynchronous operations.
+        :param serial_dev: Serial device identifier.
+        """
+        _Cube.__init__(self, loop, serial_dev)
```

Comment:
```suggestion
    def __init__(self, serial_dev: str) -> None:
        """Initialize the KPA101 driver.

        :param loop: Event loop for asynchronous operations.
        :param serial_dev: Serial device identifier.
        """
        _Cube.__init__(self, serial_dev)
```
Event loop no longer a paramter since #25.